src/main/java/net/jeqo/gizmo/data/UpdateChecker.java

```diff
@@ -1,34 +1,34 @@
 package net.jeqo.gizmo.data;
 
 import net.jeqo.gizmo.Gizmo;
+import net.jeqo.gizmo.Utils.ColourUtils;
 import org.bukkit.Bukkit;
 import org.bukkit.entity.Player;
 import org.bukkit.event.EventHandler;
 import org.bukkit.event.Listener;
 import org.bukkit.event.player.PlayerJoinEvent;
-import org.bukkit.plugin.java.JavaPlugin;
 
 import java.io.IOException;
 import java.io.InputStream;
 import java.net.URL;
 import java.util.Scanner;
 import java.util.function.Consumer;
 
-import static net.jeqo.gizmo.data.Placeholders.gizmoPrefix;
-
 public class UpdateChecker implements Listener {
 
-    private final JavaPlugin plugin;
+    private final Gizmo plugin;
     private final int resourceId;
-    public UpdateChecker(JavaPlugin plugin, int resourceId) {
+
+    private final ColourUtils colourUtils = new ColourUtils();
+
+    public UpdateChecker(Gizmo plugin, int resourceId) {
         this.plugin = plugin;
         this.resourceId = resourceId;
     }
 
-
     // Check for updates
     public void getVersion(final Consumer<String> consumer) {
-        Bukkit.getScheduler().runTaskAsynchronously(this.plugin, () -> {
+        Bukkit.getScheduler().runTaskAsynchronously(plugin, () -> {
```

Comment:
We should be using instancing to access the plugin instance rather than just defining the plugin outright without the `this` prefix. It can lead to some messy issues otherwise